www/src/components/Breadcrumbs.js

```diff
@@ -1,4 +1,6 @@
-import { createContext, useContext, useMemo, useState } from 'react'
+import {
+  createContext, useContext, useMemo, useState,
+} from 'react'
```

Comment:
@floreks Does `@pluralsh/eslint-config-typescript` set max line width? It looks like lines are broken quite fast and in my opinion this makes code a bit harder to read. Can we change that?

@pluralsh/frontend What do you think?

Reply:
This is not related to the PR itself. Pinged you just for discussion.

Reply:
There are basically no custom rules since it only extends the airbnb configs. This might also be another rule that checks a number of variables/method parameters. I think the default is 3 and that's why it breaks at 4.